internal/pkg/deploy/cloudformation/stack/env_integration_test.go

```diff
@@ -292,6 +292,38 @@ network:
 			}(),
 			wantedFileName: "template-with-importedvpc-flowlogs.yml",
 		},
+		"generate template with public application load balancer with out a default security policy": {
+			input: func() *deploy.CreateEnvironmentInput {
+				rawMft := `name: test
+type: Environment
+http:
+  public:
+    certificates:
+      - cert-1
+    security_policy: ELBSecurityPolicy-FS-1-1-2019-08`
```

Comment:
I don't see `ELBSecurityPolicy-FS-1-1-2019-08` in any of the updated expected CFN template file 🤔

Reply:
With the updated changes you can find `ELBSecurityPolicy-FS-1-1-2019-08` at `internal/pkg/deploy/cloudformation/stack/testdata/environments/template-with-imported-certs-sslpolicy-custom-empty-security-group.yml`